src/js/components/List/stories/Order.js

```diff
@@ -16,7 +22,12 @@ export const Order = () => {
   return (
     <Grommet theme={grommet} role="application">
       <Box align="center" pad="large">
-        <List data={ordered} onOrder={setOrder} />
+        <List
+          itemProps={{ 0: { background: 'pink' } }}
```

Comment:
I think adding `itemProps` and `defaultItemProps` to this story sort of distract from the focus of `onOrder`. I'm thinking we should just leave this story as was. We've got the unit test to cover snapshot testing.

Reply:
yes sorry that was my intentions to take this out once got approved